docs/api/IoT.js-API-Stream.md

```diff
@@ -11,184 +11,466 @@ The following shows stream module APIs available for each platform.
 | writable.end | O | O | O |
 | writable.write | O | O | O |
 
-### Contents
-
-- [Stream](#stream)
-- [Class: Stream](#class-stream)
-    - [Constructor](#constructor)
-        - [`new Stream()`](#new-stream)
-- [Class: Stream.Readable](#class-streamreadable)
-    - [Readable Constructor](#readable-constructor)
-        - [`new Stream.Readable()`](#new-streamreadable)
-    - [Prototype Functions](#prototype-functions)
-        - [`readable.isPaused()`](#readableispaused)
-        - [`readable.pause()`](#readablepause)
-        - [`readable.read([size])`](#readablereadsize)
-        - [`readable.resume()`](#readableresume)
-        - [`readable.push(chunk)`](#readablepushchunk)
-    - [Events](#events)
-        - [`'close'`](#close)
-        - [`'data'`](#data)
-        - [`'end'`](#end)
-        - [`'error'`](#error)
-        - [`'readable'`](#readable)
-- [Class: Stream.Writable](#class-streamwritable)
-    - [Writable Constructor](#Writable-constructor)
-        - [`new Stream.Writable([options])`](#new-streamWritableoptions)
-    - [Prototype Functions](#prototype-functions-1)
-        - [`writable.end([chunk][, callback])`](#writableendchunk-callback)
-        - [`writable.write(chunk[, callback])`](#writablewritechunk-callback)
-        - [`writable._readyToWrite()`](#writable_readytowrite)
-        - [`writable._write(chunk, callback, onwrite)`](#writable_writechunk-callback-onwrite)
-    - [Events](#events-1)
-        - [`'drain'`](#drain)
-        - [`'error'`](#error-1)
-        - [`'finish'`](#finish)
-- [Class: Stream.Duplex](#class-streamduplex)
-
 # Stream
 
-The stream module provides a base API that makes it easy to build objects that implement the stream interface.
+A stream is an abstract interface for working with streaming data.
+Streams can be readable, writable, or both (duplex). All streams
+are instances of EventEmitter.
+
+## Class: Stream
+
+### new Stream()
+
+**This method is only for implementing a new stream type.**
+
+This function registers the common properties of all streams.
+
+**Example**
+
+```js
+var Stream = require('stream').Stream;
+
+var stream = new Stream();
+
+stream.read = function () {
+  // A custom read function.
+}
+```
+
+
+## Class: Stream.Readable
+
+Readable streams are an abstraction for a *data source* from
```

Comment:
Isn't it correct to say 'Readable stream is~' ? The counterpart begins with singular like "Writable stream is an abstraction ....".